packages/ckeditor5-font/src/ui/colortableview.ts

```diff
@@ -71,6 +72,12 @@ export default class ColorTableView extends View {
 	 */
 	public documentColors: DocumentColorCollection;
 
+	/**
+	 * Color picker.
+	 *
+	 */
+	public _colorPickerView?: ColorPickerView;
```

Comment:
Your name indicates a private/protected variable while you marked it as a public, so what is it going to be? 😉